### PR TITLE
Fix reference to BPF instructions

### LINKED ISSUE #312: BPF Load Program failing
It looks like on Kernel 4.13, it's failing to load the BPF program with the following error from the verifier:
`jump out of range from insn 1 to 2`

### PR DESCRIPTION
**Pull request by sargun**
Fixes #312

### CODE REVIEW

File: bpf_linux.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ func loadSimpleBpf(progType BpfProgType, ret uint32) (int, error) {
 	attr := BPFAttr{
 		ProgType: uint32(progType),
 		InsnCnt:  uint32(len(insns)),
-		Insns:    uintptr(unsafe.Pointer(&insns)),
+		Insns:    uintptr(unsafe.Pointer(&insns[0])),
```
Comment (fcrisciani) [inline bpf_linux.go]: I think would be cleaner to remove the second instruction (line 36), the result should be the same

Comment (sargun) [reply bpf_linux.go]: See the bug.

**Comment by fcrisciani** [comment]
LGTM, fixes: https://github.com/vishvananda/netlink/issues/312

**Comment by aboch** [comment]
Thanks @sargun and @fcrisciani

**Pull request merged**